DigitalLearningSolutions.Web/Controllers/LearningPortalController/SelfAssessment.cs

```diff
@@ -962,6 +990,27 @@ public IActionResult VerificationSummary(int selfAssessmentId)
             return View("SelfAssessments/VerificationSummary", model);
         }
 
+        [Route("/LearningPortal/SelfAssessment/{selfAssessmentId:int}/{vocabulary}/Verification/{candidateAssessmentSupervisorId}/{selfAssessmentResultId}/Resend")]
+        public IActionResult ResendSupervisorVerificationRequest(int selfAssessmentId, string vocabulary, int candidateAssessmentSupervisorId, int selfAssessmentResultId)
+        {
+
+            frameworkNotificationService.SendResultVerificationRequest(
+                candidateAssessmentSupervisorId,
+                selfAssessmentId,
+                1,
+                User.GetCandidateIdKnownNotNull(),
+                selfAssessmentResultId
+            );
+            return RedirectToAction("SelfAssessmentOverview", new { selfAssessmentId, vocabulary });

```

Comment:
This should redirect to action "ReviewConfirmationRequests" rather than "SelfAssessmentOverview".